open_spiel/games/chess/chess_board.h

```diff
@@ -68,7 +68,7 @@ enum class PieceType : int8_t {
 
 static inline constexpr std::array<PieceType, 6> kPieceTypes = {
     {PieceType::kKing, PieceType::kQueen, PieceType::kRook, PieceType::kBishop,
-     PieceType::kKnight, PieceType::kPawn}};
+        PieceType::kKnight, PieceType::kPawn}};
```

Comment:
Please do not change formatting of existing code, it makes it slightly harder to review.